include/translatable-object-with-types-trait.php

```diff
@@ -14,36 +14,49 @@
 trait PLL_Translatable_Object_With_Types_Trait {
 
 	/**
-	 * Returns SQL query that fetches the IDs of the objects without language.
+	 * Fetches the IDs of the objects without language.
 	 *
 	 * @since 3.4
+	 * @since 3.7 Returns the result of the query instead of the SQL query only.
 	 *
 	 * @param int[] $language_ids List of language `term_taxonomy_id`.
 	 * @param int   $limit        Max number of objects to return. `-1` to return all of them.
 	 * @param array $args         An array of translated object types.
-	 * @return string
+	 * @return string[]
 	 *
 	 * @phpstan-param array<positive-int> $language_ids
 	 * @phpstan-param -1|positive-int $limit
 	 * @phpstan-param array<string> $args
 	 */
 	protected function get_objects_with_no_lang_sql( array $language_ids, $limit, array $args = array() ) {
```

Comment:
```suggestion
	protected function get_raw_objects_with_no_lang( array $language_ids, $limit, array $args = array() ) {
```
No it's method business whether or not a custom SQL query is made.